content/src/filings/annual-report.md

```diff
@@ -5,27 +5,31 @@ agency: New Jersey Division of Revenue and Enterprise Services
 id: ANNUAL_FILING
 callToActionLink: https://www.njportal.com/DOR/AnnualReports/Business?sessionType=AnnualReport
 callToActionText: File Your Annual Report
+summaryDescriptionMd: >-
+  Every business in New Jersey must file an `annual report.|tax-annual-report` The annual report confirms that
+  your contact information and business address are up to date.
 ---
-Every business in New Jersey must file an `annual report.|tax-annual-report` The annual report confirms that your contact information and business address are up to date.
 
-- - -
+## Application Requirements
 
-### Application Requirements
+- Business name
+- Business mailing address
+- `EIN|ein`
+- `Registered agent's|registered-agent` name and contact information
+- Names and addresses of all business partners
+- Applicable fees paid
 
-* Business name
-* Business mailing address
-*  `EIN|ein` 
-*  `Registered agent's|registered-agent` name and contact information
-* Names and addresses of all business partners
-* Applicable fees paid
+:::greenBox
+**Once completed, you will have:**
 
-> **Once completed, you will have:**
->
- > * Confirmation of submission of your annual report
->
+- Confirmation of submission of your annual report
 
-> **What if I do not complete my report?**
->
-> After two missed annual reports, you may lose your authority to do business in New Jersey, your `Business Registration Certificate|business-reg-certificate` may be invalidated, and you could lose access to state services, including funding opportunities and licenses.
->
-> To be reinstated, businesses will need to file and pay for past reports as well as pay a reinstatement fee.
\ No newline at end of file
+:::
```

Comment:
Extra ::: ?

Reply:
No, this is valid; it's for the section above. It just can't go directly beneath the text above because it's a list, and markdown lists need newlines (or `EOF`) above and below.